middle_end/semantics_of_primitives.ml

```diff
@@ -84,7 +84,24 @@ let for_primitive (prim : Lambda.primitive) =
   | Plslbint _
   | Plsrbint _
   | Pasrbint _
-  | Pbintcomp _ -> No_effects, No_coeffects
+  | Pbintcomp _
+  | Pnegint63
+  | Paddint63
+  | Psubint63
+  | Pmulint63
+  | Pdivint63
+  | Pmodint63
```

Comment:
mod and div have effects (division by zero). You should take care if the pr https://github.com/ocaml/ocaml/pull/702 is merged before this one to apply the same changes here.